concent_api/core/models.py

```diff
@@ -485,3 +493,478 @@ def clean(self):
+            raise ValidationError({
+                'StoredTaskToCompute': 'StoredTaskToCompute model has not same fields as golem_messages.message.tasks.TaskToCompute()'
+            })
+
+    def clean(self):
+        if not len(self.requestor_id) == GOLEM_PUBLIC_KEY_LENGTH or not isinstance(self.requestor_id, bytes):
+            raise ValidationError({
+                'requestor_id': f'requestor_id must be bytes instance and must have length equal to {GOLEM_PUBLIC_KEY_LENGTH}'
+            })
+
+        if not len(self.requestor_public_key) == GOLEM_PUBLIC_KEY_LENGTH or not isinstance(self.requestor_public_key, bytes):
+            raise ValidationError({
+                'requestor_public_key': f'requestor_public_key must be bytes instance and must have length equal to {GOLEM_PUBLIC_KEY_LENGTH}'
+            })
+
+        if not len(self.requestor_ethereum_public_key) == GOLEM_PUBLIC_KEY_LENGTH or not isinstance(self.requestor_ethereum_public_key, bytes):
+            raise ValidationError({
+                'requestor_ethereum_public_key': f'requestor_ethereum_public_key must be bytes instance and must have length equal to {GOLEM_PUBLIC_KEY_LENGTH}'
+            })
+
+        if not len(self.provider_id) == GOLEM_PUBLIC_KEY_LENGTH or not isinstance(self.provider_id, bytes):
+            raise ValidationError({
+                'provider_id': f'provider_id must be bytes instance and must have length equal to {GOLEM_PUBLIC_KEY_LENGTH}'
+            })
+
+        if not len(self.provider_public_key) == GOLEM_PUBLIC_KEY_LENGTH or not isinstance(self.provider_public_key, bytes):
+            raise ValidationError({
+                'provider_public_key': f'provider_public_key must be bytes instance and must have length equal to {GOLEM_PUBLIC_KEY_LENGTH}'
+            })
+
+        if not len(self.provider_ethereum_public_key) == GOLEM_PUBLIC_KEY_LENGTH or not isinstance(self.provider_ethereum_public_key, bytes):
+            raise ValidationError({
+                'provider_ethereum_public_key': f'provider_ethereum_public_key must be bytes instance and must have length equal to {GOLEM_PUBLIC_KEY_LENGTH}'
+            })
+
+        if not self.package_hash[:4] == HASH_FUNCTION:
+            raise ValidationError({
+                'package_hash': f'package_hash should be hashed with {HASH_FUNCTION} function'
+            })
+
+        if not len(self.package_hash) == HASH_LENGTH:
+            raise ValidationError({
+                'package_hash': f'package_hash should has length equal to {HASH_LENGTH}'
+            })
+
+        if not isinstance(self.concent_enabled, bool):
+            raise ValidationError({
+                'concent_enabled': 'concent_enabled should be instance of boolean'
+            })
+
+        if not isinstance(self.price, int) or not self.price >= 0:
+            raise ValidationError({
+                'price': 'Price must be an integer and bigger than or equal 0'
+            })
```

Comment:
This validation is not necessary. You're using `PositiveIntegerField`.

Actually, the field probably does not allow `0`. Is this the right type then?